### ISSUE TITLE
Describe models' hyperparameters in the pretrained models page

### ISSUE THREAD

**Issue by rodrigonogueira4** [OpenNMT/OpenNMT-py#1403]
To start training from a pretrained model (http://opennmt.net/Models-py/) one needs to specify the hyperparameters as arguments to train.py. I could find these only for the En-De model, like below. Where are the others?

python  train.py -data /tmp/de2/data -save_model /tmp/extra \
        -layers 6 -rnn_size 512 -word_vec_size 512 -transformer_ff 2048 -heads 8  \
        -encoder_type transformer -decoder_type transformer -position_encoding \
        -train_steps 200000  -max_generator_batches 2 -dropout 0.1 \
        -batch_size 4096 -batch_type tokens -normalization tokens  -accum_count 2 \
        -optim adam -adam_beta2 0.998 -decay_method noam 
        -warmup_steps 8000 
        -learning_rate 2 \
        -max_grad_norm 0 -param_init 0  -param_init_glorot \
        -label_smoothing 0.1 -valid_steps 10000 -save_checkpoint_steps 10000 \
        -world_size 4 -gpu_ranks 0 1 2 3

**Comment by vince62s** [comment]
It is not really straight forward because the code base has changed a lot since these trainings.
Checkpoints should work.
Anyhow this EN-DE is the only transformer one I think.
